### PR TITLE
Basic support for available text resources of Dynamic Context

### PR DESCRIPTION
**Pull request by adamretter**
Needed for XQTS

### CODE REVIEW

File: src/org/exist/xquery/functions/fn/FunUnparsedText.java [hunk 1]
```diff
@@ -36,7 +38,9 @@
 import java.nio.charset.StandardCharsets;
 
 import static org.exist.xquery.FunctionDSL.*;
+import static java.nio.charset.StandardCharsets.UTF_8;
 
+@RunWith(ParallelRunner.class)
```
Comment (dizzzz) [inline src/org/exist/xquery/functions/fn/FunUnparsedText.java]: junit reference in a non-test class? Is this intentional?

File: src/org/exist/xquery/functions/fn/FunUnparsedText.java [hunk 2]
```diff
@@ -20,6 +20,7 @@
 
 package org.exist.xquery.functions.fn;
 
+import com.googlecode.junittoolbox.ParallelRunner;
```
Comment (dizzzz) [inline src/org/exist/xquery/functions/fn/FunUnparsedText.java]: in a non-@Test class?

File: src/org/exist/xquery/functions/fn/FunUnparsedText.java [hunk 3]
```diff
@@ -28,6 +29,7 @@
 import org.exist.xmldb.XmldbURI;
 import org.exist.xquery.*;
 import org.exist.xquery.value.*;
+import org.junit.runner.RunWith;
```
Comment (dizzzz) [inline src/org/exist/xquery/functions/fn/FunUnparsedText.java]: junit?

File: src/org/exist/xquery/functions/fn/FunUnparsedText.java (hunk 1 above)
Comment (adamretter) [reply src/org/exist/xquery/functions/fn/FunUnparsedText.java]: Oops, that went into the wrong class. I have fixed that now.

File: src/org/exist/xquery/functions/fn/FunUnparsedText.java (hunk 2 above)
Comment (adamretter) [reply src/org/exist/xquery/functions/fn/FunUnparsedText.java]: Oops, that went into the wrong class. I have fixed that now.

File: src/org/exist/xquery/functions/fn/FunUnparsedText.java (hunk 3 above)
Comment (adamretter) [reply src/org/exist/xquery/functions/fn/FunUnparsedText.java]: Oops, that went into the wrong class. I have fixed that now.

**Comment by dizzzz** [comment]
darn

```
    [junit] Picked up _JAVA_OPTIONS: -Xmx2048m -Xms512m
    [junit] Using temporary storage location: /tmp/org.exist.test.ExistEmbeddedServer3120504973817841409
    [junit] Running xquery.CoreTests
    [junit] Tests run: 544, Failures: 0, Errors: 2, Skipped: 62, Time elapsed: 11.218 sec
BUILD FAILED
/home/travis/build/eXist-db/exist/build/scripts/junit.xml:153: Test xquery.CoreTests failed
Total time: 11 minutes 51 seconds
```

**Comment by adamretter** [comment]
@dizzzz should be fixed now.

**Comment by dizzzz** [comment]
@adamretter resolve issue :-(

**Pull request merged**